### PR TITLE
Remove "sudo -E" runner config for linux x86_64

### LINKED ISSUE #126: Do not run everything on x86_64-unknown-linux-gnu with `sudo`
The runner configured globally for `x86_64-unknown-linux-gnu` in the cargo config should only be applied where necessary which is the `north` package and tests. There's no need to run `nstar` or others with `sudo.

### PR DESCRIPTION
**Pull request by alfonsoros88**
fixes #126

### CODE REVIEW

**Comment by flxo** [comment]
#126 proposes to find a solution to just run north as root and not everything in the workspace. Isn't there a solution for this? Using the runner feature is quite convenient, since a build invoked via `cargo run` doesn't run as root and leaves nasty root owned files.

**Comment by alfonsoros88** [comment]
> #126 proposes to find a solution to just run north as root and not everything in the workspace. Isn't there a solution for this? Using the runner feature is quite convenient, since a build invoked via `cargo run` doesn't run as root and leaves nasty root owned files.

I haven't found any nicer way to do this. Currently i'm just using the env variable:
export CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUNNER="sudo -E"

**Comment by flxo** [comment]
The runner just needs to check the crate name for `north`. Not sure what this is for the tests.

File: .cargo/linux_runner_wrapper.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+case $1 in
+    *north) sudo -E $@
+    ;;
+    *tests*) sudo -E $@
```
Comment (flxo) [inline .cargo/linux_runner_wrapper.sh:6]: Can we limit this to the tests of `north`? The tests of everything else do not need `root`.

**Review by flxo — CHANGES_REQUESTED** [review]

File: .cargo/linux_runner_wrapper.sh (hunk 1 above)
Comment (alfonsoros88) [reply .cargo/linux_runner_wrapper.sh:6]: This is running only `north` and the `tests` (the integration test one) binaries with `sudo`. No other test matches with `*tests*`. I guess, I could try to narrow the match a bit further.

**Comment by flxo** [comment]
Top. Thanks!

**Pull request merged**